recipes/gevent/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "gevent" %}
+{% set version = "1.2.2" %}
+{% set sha256 = "4791c8ae9c57d6f153354736e1ccab1e2baf6c8d9ae5a77a9ac90f41e2966b2d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - greenlet >=0.4.10
```

Comment:
I should be reading `setup.py` too before sending PRs and not rely solely on `conda skeleton` :smile: